src/native/libs/System.Security.Cryptography.Native/opensslshim.h

```diff
@@ -313,6 +313,12 @@ extern bool g_libSslUses32BitTime;
     REQUIRED_FUNCTION(CRYPTO_malloc) \
     LEGACY_FUNCTION(CRYPTO_num_locks) \
     LEGACY_FUNCTION(CRYPTO_set_locking_callback) \
+    LIGHTUP_FUNCTION(CRYPTO_THREAD_lock_new) \
+    LIGHTUP_FUNCTION(CRYPTO_atomic_add) \
+    RENAMED_FUNCTION(CRYPTO_set_mem_functions11, CRYPTO_set_mem_functions) \
```

Comment:
I'm not sure that what you've done to work around the OSSL 1.0 version of this function will work in a non-portable build from 1.1.

I believe it will try to call the non-existent function CRYPTO_set_mem_functions11 and fail to link.

But maybe I was more clever with RENAMED_FUNCTION that I'm giving myself credit for.

Reply:
I do feel we do not need to worry about 1.0 anymore .... as long as it does not break the build.

Reply:
> I do feel we do not need to worry about 1.0 anymore .... as long as it does not break the build.

Official builds are currently done on Ubuntu 16.04 with 1.0.2. Jeremy and I have had discussions on bumping to 1.1, but at the moment I think it needs to build against 1.0.2 headers.